playgame.py

```diff
@@ -116,35 +116,51 @@ def prep_logger(log_dir, verbose, log_stderr, count):
         logger.addHandler(efh)
         efh.setFormatter(formatter)
 
-def parse_args():
+def parse_args(args):
     usage = 'plays games of Hanabi using the listed players'
 
     parser = argparse.ArgumentParser(description = usage)
 
     #Positional arguments
-    parser.add_argument('players', nargs = '+',
+    parser.add_argument('players',
+                        nargs = '+',
                         help = 'the players that will play Hanabi. First 5 will play unless in tournament mode')
 
     #Optional arguments
     parser.add_argument('-s', '--seed',
-                        default = int(round(time.time()*1000)), type = int,
+                        default = int(round(time.time()*1000)),
+                        type = int,
                         help = 'a specific seed for shuffling the deck')
-    parser.add_argument('-r', '--variant', type = int, choices = [1, 2, 3],
+    parser.add_argument('-r', '--variant',
+                        type = int,
+                        choices = [1, 2, 3],
                         default = 0,
                         dest = 'variant',
                         help = 'play the selected variant')
-    parser.add_argument('-t', '--tournament', dest = 'is_tournament',
-                        action = 'store_true',
-                        help = 'run 2 player games for all combinations of players (no repeats)')
-    parser.add_argument('-v', '--verbose', dest = 'verbose',
+    parser.add_argument('-v', '--verbose',
+                        dest = 'verbose',
                         action = 'store_true',
                         help = 'log moves and game state as game is played')
-    parser.add_argument('-l', '--log_dir', dest = 'log_dir', default = None,
+    parser.add_argument('-l', '--log_dir',
+                        dest = 'log_dir',
+                        default = None,
                         help = 'save logs to file')
-    parser.add_argument('-e', '--log_stderr', dest = 'log_stderr',
+    parser.add_argument('-e', '--log_stderr',
+                        dest = 'log_stderr',
                         help = 'log errors to file')
 
-    return parser.parse_args()
+    tournament_group = parser.add_argument_group('tournament arguments')
+    tournament_group.add_argument('-t', '--tournament',
+                        dest = 'is_tournament',
+                        action = 'store_true',
+                        help = 'run 2 player games for all combinations of players (no repeats)')
```

Comment:
Does this need to be updated to reflect the new option? This might be a little clunky, but I think it makes decent sense:
```suggestion
                        help = 'run games for all combinations of "players_per_game" players (no repeats)'
```